src/main/java/com/minecolonies/coremod/colony/requestsystem/requests/AbstractRequest.java

```diff
@@ -145,7 +145,7 @@ public void setState(@NotNull final IRequestManager manager, @NotNull final Requ
             {
                 manager.getRequestForToken(getParent()).childStateUpdated(manager, getToken());
             }
-            catch (final IllegalArgumentException ex)
```

Comment:
No this should crash.
I am catching it here exactly not for this reason, if this happens please, have them contact us since this is a major bug, and catching it is a Regression, I can much easier track crashes then dum log entries that nobody fucking sees.

Reply:
@OrionDevelopment okay thanks. this is was spamming constantly on our server last night, it may still be. talk to me on discord if you need access to console / etc